File: app/src/main/java/com/xjm/xxd/dribbird/login/LoginWebViewClientCallback.java
package com.xjm.xxd.dribbird.login;

import android.support.annotation.StringRes;

import com.xjm.xxd.dribbird.account.TokenBean;

/**
 * Created by queda on 2016/12/2.
 */

public interface LoginWebViewClientCallback {

    void showLoading(String msg);
    void showLoading(@StringRes int strId);

    void hideLoading();

    void loginSuccess(TokenBean tokenBean);

    void loginFailed();
}

File: app/src/main/java/com/xjm/xxd/dribbird/login/LoginWebViewClientCallback.kt
package com.xjm.xxd.dribbird.login

import com.xjm.xxd.dribbird.account.TokenBean

/**
 * Created by queda on 2016/12/2.
 */

interface LoginWebViewClientCallback {

    fun showLoading(msg: String?)

    fun hideLoading()

    fun loginSuccess(tokenBean: TokenBean)

    fun loginFailed()
}
